admin/javascript/src/components/accordion/styles.scss

```diff
@@ -0,0 +1,53 @@
+// Accordion styles
+.accordion-group {
+	margin-top: $spacer-y;
+
+	&__title {
+		margin-bottom: 0;
+
+		a {
```

Comment:
Still think we shouldn't use tag names as selectors. BEM recommends this as well: http://getbem.com/faq/#custom-tags-for-blocks